w-near/src/lib.rs

```diff
@@ -0,0 +1,1351 @@
+/**
```

Comment:
It makes sense to split it into 2 or 3 files to make easier to compare to existing fungible-token implementation

`lib.rs` - main entrypoint
`fungible_token.rs` - NEP21 implementation
`w_near.rs` - extra logic to mint and burn

Reply:
@evgenykuzyakov I have now split this up. Granted it is split up into more files than suggested but makes for clean separation between internal methods and public. This should now make further review more straight forward but let me know what you think